packages/components/src/inputs/tests/jest/CharacterCount.test.tsx

```diff
@@ -0,0 +1,26 @@
+import { render, screen } from "@hopper-ui/test-utils";
+
+import { CharacterCount } from "../../src/CharacterCount.tsx";
+
+describe("CharacterCount", () => {
+    it("should render with default class", () => {
```

Comment:
There are 5 tests we like to have for each component. If you look in Footer.test.tsx, you will see them.
The two below are still good to keep.